### DISCUSSION TITLE
Add warning for all users when accidentally clicking into templates.

### DISCUSSION THREAD

**Discussion by MattHG** [elementor/elementor#26851] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I have a portfolio of 70+Elementor sites that are all for my clients to work once completed and live. Many of these clients insist on having Full Admin access. One big problem with Elemntor for them (not me) is that even though they get training, they only jump in now again, and they forget some simple things, such as.. DON'T CLICK THE BLUE 'SINGLE PAGE ' EDIT TEMPLATE BOX! 
They will then proceed to delete what they think is the content but is actaully the post content widget there for taking out most of the site.

### Describe the solution you'd like.

It would be great if within elementor settings, there was the abilty turn turn on 'editing template' alert. Such as 'You are about to edit a template, are you sure you want to do this?' , or at least a warning message of some sort when a user accidentally clicks to go and edit a template.

### Describe alternatives you've considered.

It could also be an option for selecting which admins have the access to edit templates. I know user roles such as editor and lower could be an option, however, many and most clients I develop sites for insist on full access, which of course is their privilege.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by pumadog** [comment]
I don't have a solution for your request, but in that situation I give customers two accounts: One for editing without danger and one with admin rights for their archive, so they have full access "just in case".

**Reply by MattHG** [reply]
Not a bad idea at all. My issue is the Admins can also create users too, and likely to not do this, or even train them properly on do's and dont's. With the amount of sites we host and manage this feature would save a lot of time and confusion for my clients who need admin rights.

**Comment by black-eye** [comment]
Yes please, absolutely essential!

**Comment by MattHG** [comment]
This is still an issue. Can be a nightmare! any feedback?
